src/sxastarter/src/assets/sass/components/layout-container/index.scss

```diff
@@ -130,3 +132,15 @@
     flex-wrap: nowrap;
   }
 }
+
+@each $breakpoint, $value in $slc-grid-breakpoints {
+  $infix: breakpoint-infix($breakpoint, $slc-grid-breakpoints);
+
+  @if ($infix != '') {
+    @media only screen and (max-width: $value) {
+      .slc#{$infix}-stack {
+        flex-direction: column;

```

Comment:
builder.io in their stack at implementation also use ` align-items: stretch` property. Do we need it in our solution?

Reply:
Looks like its good to set this property. I have added it now.